packages/monaco-graphql/src/typings/monaco.d.ts

```diff
@@ -5,118 +5,111 @@
  *  LICENSE file in the root directory of this source tree.
  */
 
```

Comment:
[Hide whitespace changes for this file](https://github.com/acao/graphiql/pull/18/files?diff=split&w=1#diff-93080d2ce6a8ca3bfb9e955bbadbcc1e)